view/frontend/web/js/model/express-pay.js

```diff
@@ -160,17 +174,74 @@ define([
                 city: addressData['city'],
                 region: {
                     region: regionName,
-                    region_code: addressData['postalCode'],
+                    region_code: addressData['state'],
                     region_id: regionId
                 },
                 region_id: regionId,
                 postcode: addressData['postalCode'],
-                country_id: addressData['countryCode'],
-                customerAddressId: null,
-                saveInAddressBook: false
+                country_id: addressData['countryCode']
             });
 
             quote.shippingAddress(newAddress);
-        }
+            this.updateSelectedShippingMethod();
+        },
+
+        /**
+         * Update the quote selected shipping method
+         *
+         * @param shippingMethod
+         */
+        updateSelectedShippingMethod: function (shippingMethod = null) {
+            let newMethod = null;
+
+            if (shippingMethod !== null) {
+                let availableMethods = shippingService.getShippingRates().filter((method) => {
+                    if (shippingMethod === null) {
```

Comment:
`shippingMethod` cannot be null here since there's a not null check just above

Reply:
Thank you for catching this, a leftover from an earlier iteration! This extra if statement has been removed